advanced_functionality/pytorch_extend_container_train_deploy_bertopic/BERTtopic_extending_container.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>Download the data from our speciailized S3 bucket to avoid broken links in the future. You have either s3://sagemaker-sample-files/datasets/text/20_newsgroups/20_newsgroups.tar.gz or s3://sagemaker-sample-files/datasets/text/20_newsgroups/20_newsgroups_bulk.tar.gz</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3819/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>